tests/unit/amo/components/TestCategoriesPage.js

```diff
@@ -1,159 +1,21 @@
-import { getFakeI18nInst } from 'tests/unit/helpers';
+import { ADDON_TYPE_EXTENSION } from 'core/constants';
+import Categories from 'amo/components/Categories';
+import { CategoriesPageBase } from 'amo/components/CategoriesPage';
+import { visibleAddonType } from 'core/utils';
 
-
-const categoriesResponse = {
-  result: [
-    {
-      application: 'android',
-      name: 'Games',
-      slug: 'Games',
-      type: ADDON_TYPE_EXTENSION,
-    },
-    {
-      application: 'android',
-      name: 'Travel',
-      slug: 'travel',
-      type: ADDON_TYPE_EXTENSION,
-    },
-  ],
-};
-
-describe('Categories', () => {
-  function render({ ...props }) {
-    const { store } = dispatchClientMetadata();
-    store.dispatch(categoriesLoad(categoriesResponse));
-
-    const baseProps = {
-      ...mapStateToProps(store.getState()),
-      dispatch: sinon.stub(),
+describe('amo/components/CategoriesPage', () => {
+  it('renders Categories', () => {
+    const params = {
+      visibleAddonType: visibleAddonType(ADDON_TYPE_EXTENSION),
     };
-
-    return shallow(
-      <CategoriesBase i18n={getFakeI18nInst()} {...baseProps} {...props} />
+    const root = shallow(
+      <CategoriesPageBase params={params} />
     );
-  }
-
-  it('dispatches setViewContext with addonType', () => {
-    const fakeDispatch = sinon.stub();
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      dispatch: fakeDispatch,
-    });
-
-    sinon.assert.calledWith(
-      fakeDispatch, setViewContext(ADDON_TYPE_EXTENSION));
-
-    // Make sure that we update the addonType when it's changed.
-    root.setProps({ addonType: ADDON_TYPE_THEME });
-    sinon.assert.calledTwice(fakeDispatch);
-    sinon.assert.calledWith(
-      fakeDispatch, setViewContext(ADDON_TYPE_THEME));
-
-    // But don't dispatch when the same addonType is used.
-    root.setProps({ addonType: ADDON_TYPE_THEME });
-    sinon.assert.calledTwice(fakeDispatch);
-  });
-
-  it('renders Categories', () => {
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      error: false,
-      loading: false,
-    });
-
-    expect(root.find('.Categories-list').childAt(0).find('.Categories-link'))
-      .toHaveProp('children', 'Games');
-    expect(root.find('.Categories-list').childAt(1).find('.Categories-link'))
-      .toHaveProp('children', 'Travel');
-  });
-
-  it('renders loading when loading', () => {
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      categories: [],
-      error: false,
-      loading: true,
-    });
-
-    expect(root.find('.Categories-loading-info'))
-      .toIncludeText('Loading');
-  });
-
-  it('renders a message when there are no categories', () => {
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      categories: [],
-      error: false,
-      loading: false,
-    });
-
-    expect(root.find('.Categories-none-loaded-message'))
-      .toIncludeText('No categories found.');
```

Comment:
This seems like a useful message to maintain if users try to load categories for an `addonType` that doesn't have categories. Did this move and I just didn't notice, or was it removed?